cli/run.go

```diff
@@ -201,7 +222,7 @@ func run(c *cli.Context) error {
 		log.Fatalf("Failed to register  CI mode, error: %s", err)
 	}
 
-	log.Infoln(colorstring.Green("Running workflow:"), runParams.WorkflowToRunID)
+	printWorkflowSteps(bitriseConfig, runParams.WorkflowToRunID)
```

Comment:
we prints the running workflow, not steps
printWorkflowSteps -> printRunningWorkflow